### ISSUE TITLE
parse_xchannel parses incorrectly

### ISSUE THREAD

**Issue by ghost** [radiotap/python-radiotap#2]
The _parse_xchannel routine is not aligning to the required 4-octet alignment per the XChannel documentation. I fixed mine by:
```
def _parse_xchannel(packet, offset):
    mod_4_offset = offset - offset % -4
    xchannel_flags, xchannel_freq, xchannel_num, xchannel_maxpower = \
        struct.unpack_from('<IHBB', packet, mod_4_offset)
    return mod_4_offset + 8, {
        'xchannel_flags' : xchannel_flags,
        'xchannel_freq' : xchannel_freq,
        'xchannel_num' : xchannel_num,
        'xchannel_maxpower' : xchannel_maxpower
    }
```
The improper alignment was causing IndexError exceptions in the _parse_mcs due to it starting at an improper offset.

**Comment by jmberg** [comment]
Can you send a pull request?

But this code hasn't been maintained for all the newer fields ... if you're up to it, lots of new fields could be added.

**Comment by jmberg** [comment]
Also, all of the other code uses

    offset = align(offset, 4)

or similar

**Comment by bcopeland** [comment]
Fixed by reporter in #3 .

**Comment by ghost** [comment]
I'd like to contribute but I have some questions on timestamp, HE, and HE-MU. Mailing list or private email?
